sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/setup.py

```diff
@@ -35,7 +35,7 @@
 
 exclude_packages = [
         'tests',
-        'examples',
+        'samples',
```

Comment:
Folder is still called examples on the branch.
Also, your MANIFEST.in should include samples + tests (look for exampke in azure-core)

Reply:
sample directory name has been changed, also updated MANIFEST.in as required